### ISSUE TITLE
Expand options have incorrect separator

### ISSUE THREAD

**Issue by petrosagg** [balena-io-modules/odata-parser#30]
Expand options should be separated by semicolons but the parser expects them to be separated by ampersands.

See testcase "5.1.2 Expand - multiple nested options" in http://docs.oasis-open.org/odata/odata/v4.01/cs01/abnf/odata-abnf-testcases.xml

```
$expand=Items($select=Quantity;$expand=Product($select=Name,Price);@c=15)
```

**Comment by Page-** [comment]
Interesting, from what I can find expand options don't even seem to be a thing before odata 4 and so it makes sense we don't match that (pretty sure they were implemented before odata 4)

**Comment by Page-** [comment]
`;` separators were implemented in #39
